extension/src-language-server/stpa.langium

```diff
@@ -24,14 +24,15 @@ entry Model:
     ('ControlStructure' controlStructure=Graph)?
     ('Responsibilities' responsibilities+=Resps*)?
     (('UCAs' allUCAs+=ActionUCAs*) | ('Context-Table' rules+=Rule*))?
+    ('DCAs' allDCAs+=DCARule*)?
     ('ControllerConstraints' controllerConstraints+=ContConstraint*)?
     ('LossScenarios' scenarios+=LossScenario*)?
     ('SafetyRequirements' safetyCons+=SafetyConstraint*)?;
 
 Rule:
     name=ID '{'
         'controlAction:' system=[Node] '.' action=[Command]
-        'type:' type=('not-provided' | 'provided' | 'too-late' | 'too-early' | 'wrong-time' | 'applied-too-long' | 'stopped-too-soon')
+        'type:' type=(NotProvided | Provided | 'too-late' | 'too-early' | 'wrong-time' | 'applied-too-long' | 'stopped-too-soon')
```

Comment:
why are there now types for NotProvided and Provided but not for any of the other UCA types?